src/shipping/handlers.ts

```diff
@@ -43,14 +46,32 @@ export const createChannelsChangeHandler = (
   triggerChange();
 };
 
+const getaddPostalCodeRules = (
+  rules: ShippingMethodFragment_postalCodeRules[]
+) =>
+  rules
+    .filter(code => !code.id)
+    .map(
+      code =>
+        ({
+          end: code.end,
+          start: code.start
+        } as ShippingPostalCodeRulesCreateInputRange)
```

Comment:
By looking at `ShippingMethodFragment_postalCodeRules` type the `code.start` is an optional field whereas `ShippingPostalCodeRulesCreateInputRange` has a required `start` type of `String`. This is either not typed properly in the graphql schema or you should handle this case appropriately by using some kind of a fallback in case `code.start` is a null.